client/src/router/index.ts

```diff
@@ -9,6 +10,7 @@ const routes = [
     name: 'Landing',
     component: () => import('@/views/landing.vue')
   },
+  { path: '/signin/:idpHint', name: 'signin', component: Signin, props: true, meta: { requiresAuth: false } },
```

Comment:
In Signin component, you don't actually use the idpHint prop, so you can probably simplify this.

Reply:
We use this idpHint to determine the path for login. It is passed from the `sbc-authentication-options`

Reply:
Hmm, the Signin component doesn't appear to use this route data at all.

Reply:
I'm not sure it works internally in the SBC Common Components. I believe it passes that hint as a prop to the signin component. Which is defined on line 47 of the SbcSignin common component. I'd have to dig a little more into their code to fully understand.

Reply:
If I understand correctly, `path` specifies the expected (optional?) route param, and then `props: true` automatically converts params to props.
You don't use the props nor pass them to the SbcSignin component so I think this code is not needed... but I could be wrong.
Don't worry too much about this now if this needs to get merged asap.